### PR TITLE
[YUNIKORN-1677] move to simple go based web server for e2e tests.

### PR DESCRIPTION
**Pull request by zhuqi-lucas**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN/
* Put link here, and add [YUNIKORN-*Jira number*] in PR title, eg. `[YUNIKORN-2] Gang scheduling interface parameters`

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Review by craigcondit — CHANGES_REQUESTED** [review]
Please put all go code in pkg/webtest/ and pkg/cmd/webtest/ .

**Comment by zhuqi-lucas** [comment]
Thank you @craigcondit for review, addressed the comments now.

File: deployments/image/webtest/Dockerfile [hunk 1]
```diff
@@ -15,23 +15,11 @@
...
 # limitations under the License.
 
-ARG ARCH=
-FROM ${ARCH}nginx:stable-alpine
-
-# Home location for all data and configs
-ENV HOME=/opt/yunikorn
+# Imagestage: use scratch base image
+FROM --platform=$TARGETPLATFORM scratch
+COPY --chown=4444:0 document/index.html /html/
```
Comment (craigcondit) [inline deployments/image/webtest/Dockerfile:20]: Use `--chown=0:0` here. There's no need to make the content owned by the web user.

**Review by craigcondit — CHANGES_REQUESTED** [review]
Minor nit, but we're getting close.

File: deployments/image/webtest/Dockerfile (hunk 1 above)
Comment (craigcondit) [reply deployments/image/webtest/Dockerfile]: Looks good. Will +1 once we get clean e2e runs (may take a while as we've got some test flakiness).

**Review by craigcondit — APPROVED** [review]
+1 LGTM. Single e2e test failure is unrelated.

**Pull request closed without merging**